formats/json/build.gradle.kts

```diff
@@ -52,11 +52,4 @@ kotlin {
     }
 }
 
-// This task should be disabled because of no need to build and publish intermediate JsWasm sourceset
-tasks.whenTaskAdded {
```

Comment:
This workaround is not needed, locally build and publish tasks works find without it, but it caused some issues locally at my side.
I can revert it, if you know the reason why it's needed

Reply:
cc @igoriakovlev maybe this workaround is not needed anymore?

Reply:
JIC, I've now additionally checked, and `./gradlew compileJsWasmMainKotlinMetadata` executed successfully, so we don't need this hack anymore.
I believe initially, this source set contained some declarations that were available in JS and wasm-js targets, but without a shared declaration, as there is no shared source set in the stdlib for JS and wasm-js.
But I don't see anything like this now.